man/mblaze.7

```diff
@@ -87,92 +87,122 @@ OpenSMTPD,
 Postfix,
 or similar.
 .Nm
-expects your mail to reside in Maildir folders.
+expects your messages to reside in maildir folders.
 .Pp
 .Nm
-operates directly on Maildir folders and doesn't
+operates directly on maildir folders and doesn't
 use its own caches or databases.
 There is no setup needed for many uses.
-All tools have been written with performance in mind.
-Enumeration of all mails in a Maildir is avoided unless necessary,
+All utilities have been written with performance in mind.
+Enumeration of all messages in a maildir is avoided unless necessary,
 and then optimized to limit syscalls.
-Parsing mail metadata is optimized to limit I/O requests.
-Initial operations on a large Maildir may feel slow, but as soon as they
+Parsing message metadata is optimized to limit I/O requests.
+Initial operations on a large maildir may feel slow, but as soon as they
 are in the file system cache, everything is blazingly fast.
-The tools are written to be memory efficient (i.e. not wasteful), but
-whole messages are assumed to fit into RAM easily (one at a time).
+The utilities are written to be memory efficient
+.Po
+i.e. not wasteful
+.Pc ,
+but whole messages are assumed to fit into RAM easily
+.Po
+one at a time
+.Pc .
 .Pp
 .Nm
-has been written from scratch and tested on a large corpus of personal mail,
-but is not actually 100% RFC-conforming
-(which is neither worth it nor desirable).
+has been written from scratch and is now well tested,
+but it is not 100% RFC-conforming
+.Po
+which is neither worth it, nor desirable
+.Pc .
 There may be issues with very old, nonconforming, messages.
 .Pp
 .Nm
-is written in portable C, using only POSIX functions (apart from a tiny
-Linux-only optimization),
+is written in portable C, using only POSIX functions
+.Po
+apart from a tiny Linux-only optimization
+.Pc ,
 and has no external dependencies.
-It supports MIME and more than 7-bit messages (everything the host
+It supports MIME and more than 7-bit messages
+.Po
+everything the host
 .Xr iconv 3
-can decode).
+can decode
+.Pc .
 It assumes you work in a UTF-8 environment.
 .Nm
-works well together with other Unix mail tools such as
+works well with other Unix utilities such as
 .Xr mairix 1 ,
 .Xr mu 1 ,
 or
 .Xr offlineimap 1 .
 .Sh EXAMPLES
 .Nm
-tools are designed to be composed together in a pipe.
+utilities are designed to be composed together in a pipe.
 They are suitable for interactive use and for scripting,
 and integrate well into a Unix workflow.
 .Pp
-For example, you could decide you want to look at all unseen mail in your
+For example, you could decide you want to look at all unseen messages in your
 INBOX, oldest first.
-.Dl mlist -s ~/Maildir/INBOX | msort -d | mscan
+.Dl mlist -s ~/maildir/INBOX | msort -d | mscan
```

Comment:
keep these dirnames upcase, this is qmail default AFAIU.

Reply:
Done.